contents/ios/IntroToIos.md

```diff
@@ -0,0 +1,105 @@
+<frontmatter>
+  title: Introduction to iOS App Development
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to iOS Development
+
+Authors: [Bryan Lew](https://github.com/blewjy)
+
+## 1. Overview: What is iOS?
+
+iOS is the mobile operating system that runs on Apple's mobile devices, most notably the iPhone and the iPad. Programs or applications that run on iOS can be downloaded officially from the App Store, and developers of iOS applications can submit their own applications to the App Store to share it with the rest of the world. 
+
+## 2. Why iOS?
+
+In the mobile world, [2 operating systems dominate the whole market share: iOS and Android](https://www.theverge.com/2017/2/16/14634656/android-ios-market-share-blackberry-2016). While Android boasts higher market share in terms of the total number of hardware devices running the operating system, iOS is no slouch in other aspects, especially those important to us sofware engineers.
+
+#### Better compatability and standardisation
+
+iOS applications are specific to only the Apple devices of the iPhone and iPad line, whereas Android runs across numerous different types of devices, including phones, tablets, watches, and more. The large variation of devices poses a compatability problem from a developer's standpoint, because now applications have to be tested and working across all these different devices, and also regularly maintained this way. Each hardware device may possibly be running their own version of Android or will only support up to a particular version, thus Android developers will have to ensure that their application is also compatible across different versions of Android.
```

Comment:
I'm not sure if there is a preference to have this in bullet points. Some visuals might make it better?